environments/production/postgresql.yml

```diff
@@ -57,25 +57,43 @@ dbs:
   form_processing:
     proxy:
       host: pgbouncer3
-      pgbouncer_host: pgbouncer3
+      pgbouncer_endpoint: pgbouncer3
```

Comment:
If you wanted to make this the pgbouncer for commcarehq_proxy you would change this line to

```
      pgbouncer_endpoint: pgbouncer9
```
See https://docs.google.com/document/d/1-bXbHcxBPoaRfzy9mBlUWOnGnlooVaUsgwn0QgrO_14/edit for the rest of the instructions.